instagram-app/src/components/PostContainer/Post.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'
+import CommentSection from '../CommentSection/CommentSection'
```

Comment:
Make sure to always check file paths after moving things around, as that was the root of your issue that we covered in stand up!